src/iio/iio.c

```diff
@@ -129,16 +129,20 @@ mraa_iio_get_channel_data(mraa_iio_context dev)
                 fd = open(buf, O_RDONLY);
                 if (fd != -1) {
                     read(fd, readbuf, 31 * sizeof(char));
+
```

Comment:
Please don't add unnecessary formatting changes, those just pollute the git history.